compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -628,8 +628,11 @@ error.cannot.attach.functions.to.abstract.object=\
 error.abstract.object.function.cannot.have.body=\
   function ''{0}'' in abstract object ''{1}'' cannot have a body
 
+error.resource.function.cannot.be.extern=\
+  resource function ''{0}'' cannot be an ''external'' function
```

Comment:
Shall we say this is not supported in this implementation? We may have to allow this in the future. Also create an issue to track this, since this is allowed in the spec.

Reply:
Updated with 8cadac7.